src/components/calcite-input/calcite-input.tsx

```diff
@@ -289,12 +297,15 @@ export class CalciteInput {
     const childEl = [
       <this.childElType
         {...attributes}
+        aria-label={this.name || this.guid}
```

Comment:
I'm not sure we want to do this since most inputs will have a `calcite-label` wrapping them.

Reply:
Without this attribute, the `accessible()` test fails.  Should I wrap all the tests in a `calcite-label` instead?  I don't think we should always assume a label is wrapped around it.

Reply:
I think it should be wrapped if that is how it is intended to be used.

Reply:
I think I'll create a follow-up issue to address label support, which will remove this aria-label and add labels around element in the tests.

Reply:
https://github.com/Esri/calcite-components/issues/1759

Reply:
[Resolved this by this fix](https://github.com/Esri/calcite-components/pull/1736/commits/384d20114ca9421fdb14972fc0dd6e39de1c4818), but I will also take a closer look at label support in #1759